e2e/src/transaction-exclusion.spec.ts

```diff
@@ -0,0 +1,129 @@
+import { beforeAll, describe, expect, it } from "@jest/globals";
+import { TransactionExclusionClient } from "./utils/utils";
+
+const transactionExclusionTestSuite = (title: string) => {
```

Comment:
As already mentioned before, this shall be an integration test inside `transaction-exclusion-api:app` module, not an end-to-end tests. This test is a sort of integration/contract test, not end2end.


We shall have, however, a true e2e test: send a Transaction to the sequencer that cannot be proven, and verify that it was delivered and available through `linea_getTransactionExclusionStatusV1 `. This will ensure that the whole e2e flow works.

Reply:
> As already mentioned before, this shall be an integration test inside transaction-exclusion-api:app module.

Still, the end2end test you will write does not invalidate the integration test inside `transaction-exclusion-api:app` they serve different purposes.

Reply:
Yes, integration tests inside `transaction-exclusion-api:app` will be added